bite-size-notes/week-0-b-test.md

```diff
@@ -0,0 +1,9 @@
+## new notes
+
+Cats are cute.
```

Comment:
This should be `Cats are the cutest.`

Reply:
I have made the necessary changes to address this comment.